nitro-protocol/test/contracts/ForceMove/challenge.test.ts

```diff
@@ -169,6 +170,9 @@ describe('challenge', () => {
       const variableParts = states.map(state => getVariablePart(state));
       const fixedPart = getFixedPart(states[0]);
       const channelId = getChannelId(fixedPart);
+      console.log("variableParts");
+      console.log(variableParts);
```

Comment:
✂️

Reply:
Removed in [`0fea82b`](https://github.com/statechannels/go-nitro/pull/508/commits/0fea82b1634e81ba892dad33ad304ae4d36db885)